blockchain/msgs_test.go

```diff
@@ -98,7 +98,7 @@ func TestBlockchainMessageVectors(t *testing.T) {
 			BlockRequest: &bcproto.BlockRequest{Height: math.MaxInt64}}},
 			"0a0a08ffffffffffffffff7f"},
 		{"BlockResponseMessage", &bcproto.Message{Sum: &bcproto.Message_BlockResponse{
-			BlockResponse: &bcproto.BlockResponse{Block: bpb}}}, "1a95010a92010a5b0a02080b1803220b088092b8c398feffffff012a0212003a20c4da88e876062aa1543400d50d0eaa0dac88096057949cfb7bca7f3a48c04bf96a20e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b85512330a0b48656c6c6f20576f726c641a0022003220c4da88e876062aa1543400d50d0eaa0dac88096057949cfb7bca7f3a48c04bf9"},
+			BlockResponse: &bcproto.BlockResponse{Block: bpb}}}, "1a93010a90010a5b0a02080b1803220b088092b8c398feffffff012a0212003a20c4da88e876062aa1543400d50d0eaa0dac88096057949cfb7bca7f3a48c04bf96a20e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b85512310a0b48656c6c6f20576f726c641a003220c4da88e876062aa1543400d50d0eaa0dac88096057949cfb7bca7f3a48c04bf9"},
```

Comment:
[Non Blocking][Question]
how are these generated?

Reply:
its just the raw protobuf encoding, it has to be generated with each change. This test is a forcing function for reviewers iirc